README.md

```diff
@@ -32,3 +32,147 @@ of EDL on this cluster is
+
+Before we get to the docker image part, we recommend running the EDL controller within a Kubernetes namespace, which provides better isolation among apps. By default, the EDL runs under namespace "paddlecloud". To create it, run the following command if you don't have it created.
+
+``` bash
+kubectl create namespace paddlecloud
+```
+
+There are 2 ways to obtain the EDL docker image:
+
+1. Directly pull the pre-built image from docker hub's paddle repo
+1. Build your own
+
+If you decide to use the pre-built image, there is nothing you need to do now, you can skip to the deployment part.
+
+To build your own docker images, run the following command:
+
+``` bash
+docker build -t yourRepoName/edl-controller .
+```
+
+This command will take the `Dockerfile`, build the EDL docker image and tag it as `yourRepoName/edl-controller`
+
+Now you want to push it to your docker hub so that Kubernetes cluster is able to pull and deploy it.
+
+``` bash
+docker push yourRepoName/edl-controller
+```
+Before deploying your EDL controller, open `edl_controller.yaml` with any text editor to change the docker image uri from `paddlepaddle/edl-controller` to `yourRepoName/edl-controller`
+
+Now let's deploy the EDL controller:
+
+``` bash
+kubectl create -f edl_controller.yaml
+```
+
+To verify the deployment, let's firstly verify the depolyment's pod is successfully created:
+
+``` bash
+kubectl get pods --namespace paddlecloud
+
+NAME                                       READY     STATUS    RESTARTS   AGE
+training-job-controller-2033113564-w80q6   1/1       Running   0          4m
+```
+Wait until you see `STATUS` is `Running`, run the following command to see controller's working log:
+
+``` bash
+kubectl logs training-job-controller-2033113564-w80q6 --namespace paddlecloud
+```
+
+when you see logs like this:
+
+``` text
+t=2018-03-13T22:13:19+0000 lvl=dbug msg="Cluster.InquiryResource done" resource="{NodeCount:1 GPURequest:0 GPULimit:0 GPUTotal:0 CPURequestMilli:265 CPULimitMilli:0 CPUTotalMilli:2000 MemoryRequestMega:168 MemoryLimitMega:179 MemoryTotalMega:1993 Nodes:{NodesCPUIdleMilli:map[minikube:1735] NodesMemoryFreeMega:map[minikube:1824]}}" stack="[github.com/paddlepaddle/edl/pkg/autoscaler.go:466 github.com/paddlepaddle/edl/pkg/controller.go:72]"
+```
+That means your EDL controller is actively working monitoring and adjusting resource distributions.
+
+## Deploying a training-job
+
+Now we have a resource typed `training-job` defined in Kubernetes and we have the EDL watching and optimizing the resource distribution, let's create a training job to see how it works.
+
+Firstly, let's create your training job's docker image, which contains training logic in `example/train_ft.py`
+
+``` bash
+cd example
+docker build -t yourRepoName/my_edl_training_job .
+```
+
+then push it to docker hub to be accessible by Kubernetes:
+
+``` bash
+docker push yourRepoName/my_edl_training_job
```

Comment:
Maybe add `git clone` command before head so that readers are not confused by `yourRepoName`

Reply:
agreed, will update with "Resource Adjustments by EDL" part and FAQ in next PR